exercises/dnd-character/example.jl

```diff
@@ -0,0 +1,37 @@
+function modifier(ability)
+    return floor((ability - 10) / 2)
+end
+
+function ability()
+    return rand(3:18)
+end
+
+mutable struct DndCharacter
+    strength::Integer
+    dexterity::Integer
+    constitution::Integer
+    intelligence::Integer
+    wisdom::Integer
+    charisma::Integer
+    hitpoints::Integer
```

Comment:
Ideally you want to avoid fields that have an abstract type (see the [Julia Performance Guide](https://docs.julialang.org/en/v1/manual/performance-tips/index.html#Avoid-fields-with-abstract-type-1)), `Int` or some other specific Integer type would be fine.